basis/io/files/windows/windows.factor

```diff
@@ -245,22 +245,12 @@ M: windows init-stdio
 : (open-append) ( path -- win32-file )
     GENERIC_WRITE OPEN_ALWAYS 0 open-file ;
 
-: open-existing ( path -- win32-file )
-    flags{ GENERIC_READ GENERIC_WRITE }
-    share-mode
-    f
-    OPEN_EXISTING
-    FILE_FLAG_BACKUP_SEMANTICS
-    f CreateFileW dup win32-error=0/f <win32-file> ;
+ALIAS: open-existing open-r/w
 
```

Comment:
Here you could go "all the way" and also replace all calls to `open-existing` with `open-r/w`.